CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+<!-- todo: add your changes here, so others will know what has changed in this new version :-) -->
```

Comment:
Please don't forget do document your changes here :-) 

You should be using the keywords: 

```
Types of changes
Added for new features.
Changed for changes in existing functionality.
Deprecated for soon-to-be removed features.
Removed for now removed features.
Fixed for any bug fixes.
Security in case of vulnerabilities
```

ref: https://keepachangelog.com/en/1.0.0/#how

Reply:
Added changelog. Please let me know if this as desired